### ISSUE TITLE
New themes or layout for calculator

### ISSUE THREAD

**Issue by niranjank2022** [niranjank2022/Cluster-of-Projects#4]
You can check the calculator here:
https://niranjank2022.github.io/Cluster-of-Projects/Simple/Calculator/

As you could see, the theme of the calculator might not be pleasing. So, try to come up with unique and creative colour themes and other design features.

**Comment by Rituraj-13** [comment]
Hey, @niranjank2022 , I would love to work on it. So, please assign this to me.

**Comment by niranjank2022** [comment]
> Hey, @niranjank2022 , I would love to work on it. So, please assign this to me.

Yes, @Rituraj-13 . I really appreciate your interest. You can work on the issue and submit a PR.

**Comment by surenkumard** [comment]
@niranjank2022 - I would like to add some features, So can you please assign it to me.

**Comment by Rituraj-13** [comment]
Hey @niranjank2022, I have made required changes to improve the theme. 
And, here is the link to my PR - **#7**
